bmq-sdk/src/test/java/com/bloomberg/bmq/it/SessionIT.java

```diff
@@ -2919,12 +2914,10 @@ public void testPushProperties() throws BMQException, IOException {
 
         BmqBroker[] brokers =
                 new BmqBroker[] {
-                    BmqBrokerContainer.createContainer(TESTED_PORT),
-                    BmqBrokerTestServer.createStoppedBroker(TESTED_PORT)
+                    BmqBrokerContainer.createContainer(TESTED_PORT)
                 };
 
-        assertTrue(brokers[0].isOldStyleMessageProperties());
-        assertFalse(brokers[1].isOldStyleMessageProperties());
+        assertFalse(brokers[0].isOldStyleMessageProperties());
```

Comment:
Same question regarding old props.
Also, we have only one type of broker here so the loop may be removed

Reply:
Replaced loop with try-with resources